### PR TITLE
Add method batchBeginsWithDDLKeyword to SQLParser

### PR DESCRIPTION
**Pull request by wolffcm**
Pursuant to ongoing FILE -batch.  I put this method into SQLParser since it seemed to belong there.

### CODE REVIEW

File: tests/frontend/org/voltdb/parser/TestSQLParser.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                + "update t set i = 70 where i > 69;"));
+
+        assertFalse(SQLParser.batchBeginsWithDDLKeyword(
+                "// create table was done earlier...\n"
+                + "update t set i = 70 where i > 69;"));
+
+        assertFalse(SQLParser.batchBeginsWithDDLKeyword(
+                "/* create table was done earlier... */\n"
+                + "update t set i = 70 where i > 69;"));
+    }
```
Comment (paulmartel) [inline tests/frontend/org/voltdb/parser/TestSQLParser.java]: Add negative test cases for near-miss non-ddl statements that could cause future confusion: load, remove, exec, import, file.

Comment (wolffcm) [inline tests/frontend/org/voltdb/parser/TestSQLParser.java]: I can certainly do this.

By the way `queryIsDDL` will return true if the first keyword is `IMPORT`.  Should we actually return false for this case, or let the server take care of it?  It seems to be the first keyword in an IMPORT CLASS statement, but I'm not clear on how that statement is used.

Comment (wolffcm) [inline tests/frontend/org/voltdb/parser/TestSQLParser.java]: IMPORT CLASS seems to be treated as DDL by the server (even if it's not supported by ad hoc DDL) so I'm tempted to make sqlcmd continue to consider IMPORT CLASS to be DDL---the error messages will be consistent.
```
1> import class org.voltdb.compiler.VoltCompiler
2> ;
AdHoc DDL contains an unsupported DDL statement: import class org.voltdb.compiler.VoltCompiler
```

**Comment by jxchinaren** [comment]
The other looks good to me.

**Pull request merged**